egs/iam/s5/local/prepare_data.sh

```diff
@@ -1,5 +1,20 @@
 #!/bin/bash
-# This script loads the IAM handwritten dataset
+
+# Copyright 2017 (Author: Chun Chieh Chang, Ashish Arora)
+
+# This script loads the IAM handwritten dataset, LOB corpora and Brown corpora.
+#    It calls local/process_data.py which  reads the downloaded data and creates following
+#    files: text, utt2spk, images.scp. It also calls utt2spk_to_spk2utt.pl which creates
+#    spk2utt from utt2spk.
+# Args:
+#  --nj: number of jobs.
+#  --dir: download directory.
+#
+#  Eg. local/prepare_data.sh --nj 20 --dir data
+#  Eg. text file: 000_a01-000u-00 A MOVE to stop Mr. Gaitskell from
+#      utt2spk file: 000_a01-000u-00 000
+#      images.scp file: 000_a01-000u-00 data/download/lines/a01/a01-000u/a01-000u-00.png
+#      spk2utt file: 000 000_a01-000u-00 000_a01-000u-01 000_a01-000u-02 000_a01-000u-03
```

Comment:
Also:
remove the variable `dir` altogether.
change `lines=$dl_dir/lines` to `lines=data/local/lines` and do the same for 
`xml`, `ascii`, `bcorpus`, and `lobcorpus`. The idea is to keep the download dir
and local extractions separate so the download dir is intact. 
@ChunChiehChang, please apply the same idea to UW3. 
Please also make sure all the original downloaded files exist in `/export/corpora5/handwriting_ocr/<IAM|UW3>` and set the --download-dir to that path when calling this script. 
Ashish, please rename dl_dir to download_dir.